plugins/europeana/record.js

```diff
@@ -29,27 +29,110 @@ function parseRecordDataFromApiResponse(response) {
       link: providerAggregation.edmIsShownAt,
       src: europeanaAggregation.edmPreview
     },
-    coreFields: lookupEntities(omitBy({
-      dcContributor: proxyData.dcContributor, // Plus rdaGr2DateOfBirth & rdaGr2DateOfDeath
-      dcCreator: proxyData.dcCreator, // Plus rdaGr2DateOfBirth & rdaGr2DateOfDeath
-      dcPublisher: proxyData.dcPublisher,
-      dcSubject: proxyData.dcSubject,
-      dcType: proxyData.dcType,
-      dcTermsMedium: proxyData.dctermsMedium
-    }, isUndefined), entities),
-    fields: lookupEntities(omitBy({
-      dcTermsCreated: proxyData.dcTermsCreated,
-      edmCountry: europeanaAggregation.edmCountry,
-      edmDataProvider: providerAggregation.edmDataProvider,
-      edmRights: providerAggregation.edmRights
-    }, isUndefined), entities),
+    coreFields: coreFields(proxyData, entities),
+    fields: extraFields(proxyData, edm, entities),
     media: aggregationMedia(providerAggregation),
     agents: edm.agents,
     concepts: edm.concepts,
     title: proxyData.dcTitle
   };
 }
 
+/**
+ * See: https://github.com/TehShrike/deepmerge#arraymerge-example-combine-arrays
+ */
+function combineMerge(target, source, options) {
+  const destination = target.slice();
+
+  source.forEach((item, index) => {
+    if (typeof destination[index] === 'undefined') {
+      destination[index] = options.isMergeableObject(item) ? merge(emptyTarget(item), item, options) : item;
+    } else if (options.isMergeableObject(item)) {
+      destination[index] = merge(target[index], item, options);
+    } else if (target.indexOf(item) === -1) { // Only add values not yet present
+      destination.push(item);
+    }
+  });
+  return destination;
+}
+
+/**
+ * Retrieves the "Core" fields which will always be displayed on record pages.
+ *
+ * @param {Object[]} proxyData All core fields are in the proxyData.
+ * @param {Object[]} entities Entities in order to perform entity lookups
+ * @return {Object[]} Key value pairs of the metadata fields.
+ */
+function coreFields(proxyData, entities) {
+  return lookupEntities(omitBy({
+    dcContributor: proxyData.dcContributor, // Plus rdaGr2DateOfBirth & rdaGr2DateOfDeath
```

Comment:
What exactly does this comment and the same one on the following line mean? If irrelevant, remove them.

Reply:
On the Ruby portal we show additional information for the agents, it's out of scope for this PR.